docs/source/basic/blame.rst

```diff
@@ -0,0 +1,91 @@
+Blame
+=====
```

Comment:
I noticed some modules add links to the libgit2 API docs and some, like `blame`, don't. Is there a reason for that? Or could you add the links to these other modules too?

Reply:
I will add them soon.

Reply:
Awesome, thanks!